packages/ui-components/src/components/FormHint/FormHint.component.tsx

```diff
@@ -5,14 +5,38 @@
 
 import React, { ReactNode } from "react"
 
-const baseStyles = `
+const formHintBaseStyles = `
   jn-text-xs
   jn-mt-1
 `
 
 type FormHintVariant = "help" | "error" | "success"
 
-const variantStyles = (variant: FormHintVariant) => {
+export interface FormHintProps extends React.ComponentPropsWithoutRef<"div"> {
+  /**
+   * The content to render as a hint for a form element.
+   * If children are provided, they will take precedence over text.
+   */
+  children?: ReactNode
+
+  /**
+   * The text to render as a hint for a form element.
+   * Overridden by children, if provided.
+   */
+  text?: ReactNode | string
```

Comment:
That's a great point, thanks for the catch! 🚀 I agree that the type here should only be `string` really, since we allow `children` for the rest. However, currently it's `children: PropTypes.node and `text PropTypes.node` for some reason 🤷‍♀️. 

Only thing I did here was allow `string` as an additional type, couse I think it makes sense to specify `string` explicitly for `text`. Since node includes string, it doesn't introduce anything new. And it can be the transition point for us removing `node` in the future, what do you think?

We can't remove `node` for now, since it would be a breaking change. I added this to our improvement epic. We should remove `node` and only allow `string` here.